c_src/shmex/shmex.c

```diff
@@ -23,6 +23,26 @@ int load(ErlNifEnv *env, void **priv_data, ERL_NIF_TERM load_info) {
   return 0;
 }
 
+static ERL_NIF_TERM export_realloc(ErlNifEnv *env, int argc,
```

Comment:
This looks exactly like set_capacity, but the docs for that say `If the capacity is smaller than the current size, data will be discarded and size modified `